log4j-layout-jackson-json-template/src/main/resources/JsonLayout.json

```diff
@@ -0,0 +1,27 @@
+{
+  "instant": {
+    "epochSecond": "${json:timestamp:epoch:divisor=1e9,integral}",
+    "nanoOfSecond": 0
```

Comment:
I don't have a directive for `nanoOfSecond`, yet. Would you rather want to have it in the first release?

Reply:
@rgoers, any ideas?

Reply:
Sorry for not responding sooner.  I am fine with skipping features so long as we are not trying to be completely compatible with the existing JSONLayout.